satviz-consumer/src/main/java/edu/kit/satviz/consumer/cli/HeatmapColorsType.java

```diff
@@ -29,11 +31,24 @@ public HeatmapColors convert(ArgumentParser parser, Argument arg, String value)
     }
     String[] stringArray = value.split(":");
     HeatmapColors heatmapColors = new HeatmapColors();
-    heatmapColors.setFromColor(Integer.parseInt(stringArray[0].substring(1)));
-    heatmapColors.setToColor(Integer.parseInt(stringArray[1].substring(1)));
+    heatmapColors.setHotColor(intToColor(Integer.parseInt(stringArray[0].substring(1))));
```

Comment:
Hier auch das Problem mit from = cold / to = hot. 
Zudem kann man einfach die API von Color verwenden, um das direkt vom String aus zu parsen. Ich weiß nicht, ob die Verwendung so zu 100% stimmt, aber das ist zumindest der Ansatz.
```suggestion
    heatmapColors.setColdColor(Color.web(stringArray[0]));
```